package.json

```diff
@@ -121,5 +121,7 @@
     "typecheck": "tsc --build src/tsconfig.json",
     "report-coverage": "codecov -f coverage/coverage-final.json",
     "version": "make clean build"
-  }
+  },
+  "private": true,
+  "workspaces": ["frontend-shared"]
```

Comment:
- Yes, the name does not include scope
- This bit of code will be removed once this moves to its own repo much later